applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -147,7 +147,7 @@ int hw_codec_volume_adjust(int8_t adjustment_db)
 	prev_volume_reg_val = new_volume_reg_val;
 
 	/* This is rounded down to nearest integer */
-	LOG_INF("Volume: %ddB", (new_volume_reg_val / 2) - MAX_VOLUME_DB);
+	LOG_DBG("Volume: %" PRId32 " dB", (new_volume_reg_val / 2) - MAX_VOLUME_DB);
```

Comment:
What does this mean?

Reply:
This is the standard C way of printing fixed-width values, see e.g. https://www.ibm.com/docs/en/zos/2.3.0?topic=files-inttypesh-io-format-integer-types